api/Element.json

```diff
@@ -4353,7 +4353,7 @@
               ]
             },
             "ie": {
-              "version_added": "5.5"
+              "version_added": "9"
```

Comment:
Makes sense that various *NS methods would be added together in IE9, when they were needed for SVG. Previously the `getElementsByTagNameNS` data matched `getElementsByTagName`, and was probably just copied at some point in the past.